### PR TITLE
Fix SXG warning

### PR DESCRIPTION
**Pull request by sebastianbenz**
warning: 199 - "inner != outer; fallback url https://amp.dev/ != https://amp.dev"

### CODE REVIEW

File: platform/lib/routers/packager.js [hunk 1]
```diff
@@ -58,11 +58,14 @@ const packager = (request, response, next) => {
...
   }
+  // sign https://amp.dev instead of https://amp.dev/
+  const urlToSign = request.url === '/' ? '' : request.url;
   // Hard-code amp.dev as it has to match the cert
   const searchParams = new URLSearchParams({
-    sign: 'https://amp.dev' + request.url,
+    sign: 'https://amp.dev' + urlToSign,
   }).toString();
   const url = `/priv/doc?${searchParams}`;
+  console.log('sign', url);
```
Comment (ithinkihaveacat) [inline platform/lib/routers/packager.js]: Is the added `console.log` intentional?

**Comment by ithinkihaveacat** [comment]
LGTM, though haven't checked to see if `amppkg` adds that slash back on.

/cc @twifkak

File: platform/lib/routers/packager.js (hunk 1 above)
Comment (sebastianbenz) [reply platform/lib/routers/packager.js]: nope - fixed

**Comment by twifkak** [comment]
This fixes the warning for https://amp-dev.cdn.ampproject.org/c/s/amp.dev but will it create a warning for https://amp-dev.cdn.ampproject.org/c/s/amp.dev/ ? Hopefully you can support both, but if you have to choose one, I guess I'd go for whichever search considers canonical.

**Pull request merged**